contracts/0.8.9/oracle/AccountingOracle.sol

```diff
@@ -522,11 +548,13 @@ contract AccountingOracle is BaseOracle {
             );
         }
 
-        ILegacyOracle(LEGACY_ORACLE_POSITION.getStorageAddress()).handleConsensusLayerReport(
-            data.refSlot,
-            data.clBalanceGwei * 1e9,
-            data.numValidators
-        );
+        if (LEGACY_ORACLE != address(0)) {
```

Comment:
how can it be zero? I'd prefer not to have this check if it's not 100% necessary

Reply:
I'm not sure too actually. It adds complexity due to the requirement to have LegacyOracle deployed and initialized in tests and deployment from scratch. And gives kind of feeling that `LidoOracle.handleConsensusLayerReport` is kind of tested. Although it get tested indeed only during tests after upgrade.

But I guess it's likely to be cheaper to forbid zero address than to discuss it.